labels.mss

```diff
@@ -53,18 +53,34 @@
     text-character-spacing: 2;
   }
 }
-#city[capital='yes'][zoom>=5][zoom<=10],
+
 #city[zoom>=5][zoom<=10],
 #place_low[type='city'][zoom>=7][zoom<=10],
 #place_low[type='town'][zoom>=9][zoom<=10] {
   shield-file: url('icon/place/town.svg');
-  #city[capital='no'],
   [type='city'] {
     shield-file: url('icon/place/city.svg');
+    shield-line-spacing: -2;
+    shield-text-dx: 6;
+  }
+  [type='hc'],[type='ambassade'],[type='capitale'] {
+    shield-fill: @city_text;
+    shield-face-name: @bold;
+    shield-halo-fill: @halo;
+    shield-size: 12;
+    shield-text-dx: 6;
+  }
+  [type='hc'],[type='ambassade'] {
+    shield-file: url('icon/place/ambassade.svg');
+    shield-allow-overlap: true;
   }
-  [capital='yes'] {
+  [type='capitale'] {
     shield-file: url('icon/place/capital.svg');
   }
+  [type='consulat'],[type='consulat_general'],[type='rp'],[type='bureau'] {
```

Comment:
A few things I'd suggest:
- extrapolate categories right in SQL to keep the cartocss and XML simple/light with `CASE`, something like:

    `CASE WHEN type IN ('consulat', 'consulat_general') THEN 'intermediate' ELSE type END AS type`

- keep cartocss in english so we don't mix, and thus use the SQL to translate the categories if we want to keep the categories in French in the files

Reply:
try to lauch a make import_city before?

Reply:
so we have capital, embassy, intermediate

Reply:
and intermediate.svg and embassy.svg